main.go

```diff
@@ -43,45 +40,30 @@ func main() {
 	// sensitive fields are omitted from config.String().
 	log.Event(ctx, "config on startup", log.INFO, log.Data{"config": cfg})
 
-	elasticHTTPClient := rchttp.NewClient()
+	elasticHTTPClient := dphttp.NewClient()
 	elasticsearch := elasticsearch.NewElasticSearchAPI(elasticHTTPClient, cfg.ElasticSearchAPIURL, cfg.SignElasticsearchRequests)
-	_, status, err := elasticsearch.CallElastic(context.Background(), cfg.ElasticSearchAPIURL, "GET", nil)
-	if err != nil {
-		log.Event(ctx, "failed to start up, unable to connect to elastic search instance", log.FATAL, log.Error(err), log.Data{"http_status": status})
-		os.Exit(1)
-	}
 
-	producerChannels := kafka.CreateProducerChannels()
-	hierarchyBuiltProducer, err := kafka.NewProducer(ctx, cfg.Brokers, cfg.HierarchyBuiltTopic, cfg.KafkaMaxBytes, producerChannels)
+	// producerChannels := kafka.CreateProducerChannels()
+	// hierarchyBuiltProducer, err := kafka.NewProducer(ctx, cfg.Brokers, cfg.HierarchyBuiltTopic, cfg.KafkaMaxBytes, producerChannels)
+	// exitIfError(ctx, err, "error creating kafka hierarchyBuiltProducer")
+	// hierarchyBuiltProducer.Channels().LogErrors(ctx, "error received from hierarchy built kafka producer, topic: "+cfg.HierarchyBuiltTopic)
```

Comment:
Remove the commented code?

Reply:
Lines 46-49 are not needed as these are replaced with line 51+. I should reinstate line 49 after I've created a new kafka producer for hierarchy topic